### PR TITLE
Climatic menu

### LINKED ISSUE #1819: The climatic menu
The new climatic menu will have a different structure to the existing one.  We would like it to be the same (in one column) of the main menu.

So the main items will be File, Organise, Describe and Model.  There may be more than one menu of any of these "types" if we need, e.g. Model1, Model2.

There is already a Climatic > File, with just one item so far.  This is fine.
The current **Climatic > Manage** changes to **Climatic > Organise**.
Add a Climatic > Describe.  
The current Climatic > Display Daily goes into Climatic > Describe.
The Current Climatic > Summary will go into the new Climatic > Organise
The current Climatic > Events will go into the new Climatic > Organise.  This will give a third level.  That's OK at this stage.
More to come!

### PR DESCRIPTION
**Pull request by maxwellfundi**
Fixes #1819

### CODE REVIEW

File: instat/frmMain.vb [hunk 1]
```diff
@@ -332,55 +332,55 @@ Public Class frmMain
...
         dlgExtremes.ShowDialog()
     End Sub
 
-    Private Sub mnuClimaticEventsSpells_Click(sender As Object, e As EventArgs) Handles mnuClimaticEventsSpells.Click
+    Private Sub mnuClimaticEventsSpells_Click(sender As Object, e As EventArgs)
         dlgSpells.ShowDialog()
     End Sub
 
-    Private Sub mnuClimaticEventsWaterBalance_Click(sender As Object, e As EventArgs) Handles mnuClimaticEventsWaterBalance.Click
+    Private Sub mnuClimaticEventsWaterBalance_Click(sender As Object, e As EventArgs)
```
Comment (dannyparsons) [inline instat/frmMain.vb]: Can you also remove these old subs now because they don't handle any event.

**Comment by dannyparsons** [comment]
I've merged the other one and now this has a conflict. Can you look at resolving it? If you can't then I can have a go at resolving it.

**Comment by maxwellfundi** [comment]
Is it something easy to do? Will it take lots of your time?
Seems i cant really get it fixed, maybe closing this and redoing all the work.

**Comment by dannyparsons** [comment]
Is it much work to redo? If it is I can try to look at resolving it.

**Comment by maxwellfundi** [comment]
Its re-Arranging menus and stuff. I think i can just redo if this will take a lot of time for you. I thought its easy for people with write access.

**Comment by dannyparsons** [comment]
Let me have a quick check first.

**Comment by maxwellfundi** [comment]
Okay, great. Let me know how it goes.

**Comment by dannyparsons** [comment]
I've tried but I can't do it easily. I think the best is just to redo it, unfortunately. Make sure you have the up to date code for the new changes.

**Comment by maxwellfundi** [comment]
okay. Thanks

**Comment by maxwellfundi** [comment]
Closing this to do a new pull with same changes

**Pull request closed without merging**